src/tests/functional/navigation_tests.ts

```diff
@@ -116,6 +116,39 @@ export class NavigationTests extends TurboDriveTestCase {
     this.assert.equal(await this.pathname, "/src/tests/fixtures/one.html")
     this.assert.equal(await this.visitAction, "restore")
   }
+
+  async "test skip link with hash-only path scrolls to the anchor"() {
+    await this.clickSelector('a[href="#main"]')
+
+    const scrolledToMain = await this.isScrolledToSelector("#main")
+    this.assert.equal(await this.pathname, "/src/tests/fixtures/navigation.html")
+    this.assert.equal(await this.hash, "#main")
+    this.assert.notOk(await this.changedBody, "does not reload page")
+    this.assert.ok(scrolledToMain)
+  }
+
+  async "test skip link with hash-only path moves focus and changes tab order"() {
+    await this.clickSelector('a[href="#main"]')
+
+    await this.pressTab()
+    const activeElement = await this.getActiveElement()
+    const equalsSkippedLink = await activeElement.equals(await this.querySelector("#ignored-link"))
```

Comment:
Would it be _so_ bad to add `tabindex=-1` on `#main` and check that it's the active element?